user-guide/Cloud_Platform/TeamsBot/Microsoft_Teams_Chat_Integration.md

```diff
@@ -58,8 +60,7 @@ After you have made sure the above-mentioned prerequisites are in place, you can
 
 ## Limitations
 
-- Currently, it is not possible to retrieve the existing teams. You can only create a channel using chat integration if you know the ID of the team, which you will receive when you create the team with chat integration.
-- Currently, it is not possible to retrieve the existing channels. You can only send notifications in a channel using chat integration if you know the ID of the channel, which you will receive when you create the channel with chat integration.
+- Currently, you cannot send messages in fetched channels where the DataMiner Teams bot wasn't installed in the team. This will be indicated in the error when you try to send a channel notification. You can [manually install the DataMiner Teams bot in the team using Microsoft Teams](xref:DataMiner_Teams_bot#dataminer-teams-bot-installation) or use the create team functionality of chat integration to create another team with the DataMiner Teams bot automatically installed.
```

Comment:
```suggestion
- Currently, it is not possible to send messages in fetched channels for which the DataMiner Teams bot is not installed in the team. This will be indicated in the error when you try to send a channel notification. You can [manually install the DataMiner Teams bot in the team using Microsoft Teams](xref:DataMiner_Teams_bot#dataminer-teams-bot-installation) or use the create team functionality of chat integration to create another team with the DataMiner Teams bot automatically installed.
```

Reply:
I'll try to fix this limitation

Reply:
done